spk/kavita/src/service-setup.sh

```diff
@@ -0,0 +1,32 @@
+
+# Kavita service setup
+KAVITA="${SYNOPKG_PKGDEST}/share/Kavita"
+
+# Kavita uses custom Config and PID directories
+HOME_DIR="${SYNOPKG_PKGDEST}/share"
+CONFIG_DIR="${SYNOPKG_PKGVAR}/config"
+PID_FILE="${SYNOPKG_PKGVAR}/kavita.pid"
```

Comment:
this is the default and must not be defined here
```suggestion
```

Reply:
Thanks for this feedback.

Reply:
Should we work over time to remove these in the *arr family of apps? Many of them have the `PID_FILE` defined similarly.